qemu/tests/cfg/vioinput_enable.cfg

```diff
@@ -1,11 +1,18 @@
 - vioinput_enable: install setup image_copy unattended_install.cdrom
+    required_qemu = [2.4.0, )
+    no Win2008..sp2
     virt_test_type = qemu
     type = driver_in_use
     inputs = input1
     input_dev_bus_type_input1 = virtio
     driver_name = "vioinput"
     extra_driver_verify = "viohidkmdf hidclass.sys hidparse.sys"
     run_bg_flag = "before_bg_test"
+    del usb_devices
+    move_rate = 80
+    move_duration = 1
+    btns = "left right middle side extra"
+    scrolls = "wheel-up wheel-down"
```

Comment:
The above four lines should be moved into the mouse and the tablet variants.

Reply:
Thanks, updated~